src/components/fileUpload/resume.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+
+import { toast } from '@/components/ui/use-toast';
+import { Button } from '@/components/ui/button';
+import { axiosInstance } from '@/lib/axiosinstance';
+
+const allowedResumeFormats = ['application/pdf', 'image/png'];
+const maxResumeSize = 2 * 1024 * 1024; // 2MB in bytes
+
+const ResumeUpload = ({ user_id }: { user_id: string }) => {
+  const [selectedResume, setSelectedResume] = useState<File | null>(null);
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+
+  const handleResumeChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    if (file && allowedResumeFormats.includes(file.type)) {
+      if (file.size <= maxResumeSize) {
+        setSelectedResume(file);
+        if (file.type === 'image/png') {
+          setPreviewUrl(URL.createObjectURL(file));
+        } else {
+          setPreviewUrl(null);
+        }
+      } else {
+        toast({
+          variant: 'destructive',
+          title: 'File too large',
+          description: 'Resume size should not exceed 2MB.',
+        });
+      }
+    } else {
+      toast({
+        variant: 'destructive',
+        title: 'Invalid file type',
+        description: `Please upload a valid resume file. Allowed formats: ${allowedResumeFormats.join(', ')}`,
+      });
+    }
+  };
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+
+    if (!selectedResume) {
+      toast({
+        variant: 'destructive',
+        title: 'No Resume Selected',
+        description: 'Please select a resume before submitting.',
+      });
+      return;
+    }
+
+    const formData = new FormData();
+    formData.append('resume', selectedResume);
+
+    try {
+      const postResponse = await axiosInstance.post(
+        '/register/upload-image',
+        formData,
+        {
+          headers: {
+            'Content-Type': 'multipart/form-data',
+          },
+        },
+      );
+
+      const { Location } = postResponse.data;
+
+      const putResponse = await axiosInstance.put('/freelancer/${user_id}', {
```

Comment:
Api service

Reply:
sir rightnow there are no service files made in alpha frontend........And according to arpit sir this PR is urgent to merge rightnow

Reply:
i removed the consoles and raised the PR